post_it_backend/urls.py

```diff
@@ -16,8 +16,30 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+from rest_framework import permissions
+
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title='POST IT APIs',
+        default_version='v1',
+        description='API documentation',
+        terms_of_service='',
+        contact=openapi.Contact(email=''),
+        license=openapi.License(name='BSD Licence'),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include('api.urls')),
+
+    #DRF URLs
+    path('swagger<format>/', schema_view.without_ui(cache_timeout=0), name='schema-json'),
+    path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
```

Comment:
Can you use a more user friendly path name here instead of swagger. For example 'documentation/'